user_acceptance_tests/tests/script.rb

```diff
@@ -0,0 +1,63 @@
+require 'json'
```

Comment:
Did you intend to keep this file? It looks like it might be the old version of `tests/happy.rb`